releasenotes/notes/29385.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: feature
+area: networking
+issue:
+  - 29385
+releaseNotes:
+  - |
+    **Added** the ability for Istio to auto-generate MCS ServiceExports for non cluster-local services
```

Comment:
@howardjohn should we even add a release note at this point? I'm not sure it's something we want to advertise yet.

Reply:
Discussed offline. Release notes are not needed now. Removing.